simtools/configuration/commandline_parser.py

```diff
@@ -12,12 +12,18 @@ class CommandLineParser(argparse.ArgumentParser):
 
     Command line arguments should be given in snake_case, e.g. `input_meta`.
 
-    Methods
+    Atributes
     -------
-    initialize_default_arguments:
-        Initialize default arguments used by all applications
-    initialize_telescope_model_arguments:
-        Initialize default arguments for telescope model definitions
+    paths: bool
+        Add path configuration to list of args.
+    telescope_model: bool
+        Add telescope model configuration to list of args.
+    workflow_config: bool
+        Add workflow configuration to list of args.
+    db_config: bool
+        Add database configuration parameters to list of args.
+    job_submission: bool
+        Add job submission configuration parameters to list of args.
```

Comment:
All these are not parameters of the class init. Could you explain why you want to move this here?